tests/neg/kinds1.scala

```
@@ -0,0 +1,16 @@
+object Test {
```

This file and kinds2.scala are almost identitical, the other one could be removed.

---

`kinds2.scala` tests another error. So I think we should keep it

---

Any reason to not test `f[C]` in this file?

---

I believe because the error is not reported with this file:
> This change required some changes in the tests because errors now get reported
at different times.